.github/workflows/deploy.yml

```diff
@@ -52,6 +52,12 @@ jobs:
             - name: 📡 Push supabase migrations
               run: npx supabase db push
 
+            - name: 📡 Push supabase functions secrets
+              run: |
+                  npx supabase secrets set POSTMARK_WEBHOOK_USER=${{ secrets.POSTMARK_WEBHOOK_USER }}
```

Comment:
**suggestion:** This should be done at configuration time, not in CI

Reply:
I figured it's harmless to do it at CI time, which prevents us from forgetting, and I also documented how to do it at configuration time.
What's more, supabase can start without those secrets (they will only prevent the `postmark` function to run at call time)